### ISSUE TITLE
Get NaN loss in training

### ISSUE THREAD

**Issue by tien-ngnvan** [ouyanghaodong/DEYOv1.5#8]
Hi, thank you for your excellent work. I followed the training step-by-step DEYO v1.0, I modified the code based on [YoloV9](https://github.com/WongKinYiu/yolov9) original. After several epochs, I get the NaN value - which is hyperparameter lr0=0.0001, lrf=0.0001, optimizer=AdamW, freeze=22, cfg=gelan-c.yaml (replace DDect head which RTDetr))

![Image](https://github.com/user-attachments/assets/3085580a-f568-4f2a-b0bf-1226264904da)
